src/backend/actions/exchange-data-for-election.ts

```diff
@@ -64,7 +64,7 @@ export default proc
                     error: null,
                     wave: poll.wave,
                     seats: poll.seats,
-                    candidates: poll.candidates as string[],
```

Comment:
this data is requested [here](https://github.com/Teyvat-Collective-Network/tcn-v4/blob/hyper/prevent-election-self-voting/src/web/app/(home)/election/%5Bid%5D/page.tsx#L8) which is used in [this file](https://github.com/Teyvat-Collective-Network/tcn-v4/blob/hyper/prevent-election-self-voting/src/web/app/(home)/election/%5Bid%5D/client.tsx) to show the interface